refill-style.yaml

```diff
@@ -4186,19 +3454,19 @@ layers:
                         order: 438
                         style: building-grid
                         extrude: global.building_extrude_height
-                        color: [1.0, 1.0, 1.0]
+                        # color: global.white_color
                     lines:
                         visible: true
                         order: 439
                         style: building-lines
                         extrude: global.building_extrude_height
 
         # landuse-specific rules
-        in_park:
-            filter: { landuse_kind: [park,forest,nature_reserve,conservation,golf_course,garden] }
-            draw:
-                polygons:
-                    color: [1.0,1.0,1.0]
+        # in_park:
```

Comment:
Probably remove this commented bit (we have the idea preserved in the color theme).